### ISSUE TITLE
Launching FFXI Crashes

### ISSUE THREAD

**Issue by rfrasz4444** [Windower/Issues#798]
Just started having this problem last night.  Launching FFXI works fine but when launching it using Windower4 I get an error when POL tries to launch FFXI.  I have reinstalled Windower4 and rebooted as well as turned off all add-ons and plug-ins.  I am running Windows 10 and have not had any problems with Windower4 up to now.  I am struggling to understand what the issue is here since FFXI works fine without Windower,  Any suggestions?

Here is what is in the Event Viewer for the Application Error:

Log Name:      Application
Source:        Application Error
Date:          11/17/2015 7:10:52 AM
Event ID:      1000
Task Category: (100)
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      Home-PC
Description:
Faulting application name: pol.exe, version: 1.18.13.0, time stamp: 0x4e3bcddc
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x00000000
Faulting process id: 0xf18
Faulting application start time: 0x01d12139353864df
Faulting application path: E:\PlayOnline\SquareEnix\PlayOnlineViewer\pol.exe
Faulting module path: unknown
Report Id: ad388437-0bc0-46ee-9fad-51128572b78c
Faulting package full name: 
Faulting package-relative application ID: 
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" />
    <EventID Qualifiers="0">1000</EventID>
    <Level>2</Level>
    <Task>100</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2015-11-17T13:10:52.595854300Z" />
    <EventRecordID>355</EventRecordID>
    <Channel>Application</Channel>
    <Computer>Home-PC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>pol.exe</Data>
    <Data>1.18.13.0</Data>
    <Data>4e3bcddc</Data>
    <Data>unknown</Data>
    <Data>0.0.0.0</Data>
    <Data>00000000</Data>
    <Data>c0000005</Data>
    <Data>00000000</Data>
    <Data>f18</Data>
    <Data>01d12139353864df</Data>
    <Data>E:\PlayOnline\SquareEnix\PlayOnlineViewer\pol.exe</Data>
    <Data>unknown</Data>
    <Data>ad388437-0bc0-46ee-9fad-51128572b78c</Data>
    <Data>
    </Data>
    <Data>
    </Data>
  </EventData>
</Event>

**Comment by rfrasz4444** [comment]
To add a bit more, right after getting through POL and clicking the button to launch FFXI I see POL close off and the PC tries to launch FFXI.  At this point I get a pop up saying that Windows encountered an error and then everything is closed off.

**Comment by wildsprite** [comment]
did you recently upgrade windows 10 to its version that was released on the 10th or 11th last week? 
you can find out easily by typing winver in the search

**Comment by rfrasz4444** [comment]
I updated yesterday after all this crashing hoping that the issue may be fixed.  It is doing the same thing on the latest version now

**Comment by rfrasz4444** [comment]
@wildsprite Windows Version is Windows 10 Version 1511 (OS Build 10586.3)

**Comment by wildsprite** [comment]
okay so thus far this issue is not reproducable by the devs, I suspect if they update to the same version of windows 10 we are on though they will have it. still if it cannot be reproduced by normal means it makes it difficult to fix.

**Comment by rfrasz4444** [comment]
@wildsprite thanks for your help.  Based on what I could find the only way I could resolve the problem was a complete uninstall of windower and FFXI and a reinstall.  Everything works now but the 8 hour upgrade sucked.  We can close this off...

**Comment by z16** [comment]
I'm relatively sure this is unrelated to the Windows version. I am on the latest Windows version and I've updated regularly, I still was not able to reproduce it.

**Comment by wildsprite** [comment]
I'm not sure what to do about it. I have tried removing windower, reinstalling FFXI and no change
this didnt start till windows update installed Windows 10 Version 1511 (OS Build 10586.3)
I'm sure its some sort of library thing going on but what
also I'm pretty sure FFXI wasnt an issue because I could launch it without windower with no issues.

**Comment by rfrasz4444** [comment]
I uninstalled windower and FFXI and made sure to reboot.  I then download FFXI Ultimate Collection from the SE website (I didn't use my media from 2004).  Also had to resetup firewall rules and exclusions for virus scan.  Not sure if that will help you @wildsprite but wanted to follow-up.  It definitely sucks not to be able to use windower so I feel your pain.

**Comment by wildsprite** [comment]
@rfrasz4444 I did all of those things. my copy of the game is from the website. I tried uninstalling everything, made sure there was nothing left behind by either,  reinstalling everything. allow through my firewall. exclusions for AV...nothing worked. still crashes when I try to use windower but not when I just launch playonline by itself. 
@z16 while I can't directly blame Windows 10 Version 1511 (OS Build 10586.3) this issue did not start till it was installed so something that came with it is the issue. I'm certain of that much I just don't know what.

**Comment by rfrasz4444** [comment]
@wildsprite shitty.  One other thing that I noticed was after I reinstalled everything that it worked fine.  I was tweaking the FFXI config and Windower config and after that it crashed once.  I thought it might have had to do with screen resolution so I messed with that a bit and got it working.  Not sure if that will help but was something I ended up doing.  I think FFXI config has to be in full screen for Windower to work.  I am probably not helping but just trying to point you in some directions to try and help
